README.md

```diff
@@ -1,36 +1,19 @@
-## Chitter Challenge
+# Chitter Challenge
 
-* Challenge time: until the end of the day
-* Feel free to use google, your notes, books etc but please work on your own
-* Please raise a pull request when you start this challenge, and keep pushing updates as and when you make commits throughout the day
-* There is _no expectation_ to finish all or any of the user stories, please use this time to reflect on where you feel you are with the skill and what may support your learning.
-* If you get blocked, please reflect on what blocked you and any strategies you adopted that helped you make progress.
+For week 6 on the web we have been given the task of writing a small Twitter clone that allows users to post messages to a public stream. The user stories can be seen below.
 
-We are going to write a small Twitter clone that will allow the users to post messages to a public stream.
+## Approach
 
-## Set up
+This task started relatively easily and similar to the bookmarks pairing challenge we have been working on. After setting up the development and test database and installing the required gems I started with a feature test aimed at the first user story of seeing all messages. From here it developed to unit testing the Peep object that could retreive all peeps from the database and then creating an erb file to display them. It was a similar case for the second user story which led me to do more unit testing on the Peeps class so it could create a new peep.
 
-To setup the database:
+The user story to see the date made things a bit more complicated but as before I started with the feature test which led to me updating the database. I initially added a date column with the timestamp data type so it would automatically input the date when a new record was created. However I struggled to think of a way to test that accurately and so I decided to change the datatype to a bigint and save the time as "number of seconds since the unix epoch." I felt that gave me more flexibility when it came to formatting the date, and meant I could test it using rspecs `expect ` call.
```

Comment:
It can be tricky to write tests that rely on timing. You took the right approach by using mocking to control what time would be returned in a test. Lower down in another comment I gave some suggestions for other data types you could have used other than timestamp. 

Using a DATE or DATETIME column in postgres is preferable because you can then rely on the database to catch bad data being inserted. Using BIGINT means the DB can't detect if your code accidentally inserted numbers that are not valid timestamps for example.